src/main/java/com/openlattice/postgres/PostgresColumn.java

```diff
@@ -320,6 +320,12 @@
             .withDefault( "'-infinity'" )
             .notNull();
     public static final String                   LAST_WRITE_FIELD                  = "last_write";
+    public static final String                   LAST_SYNC_FIELD                   = "last_sync";
+    public static final PostgresColumnDefinition LAST_SYNC                         = new PostgresColumnDefinition(
+            LAST_SYNC_FIELD,
+            TIMESTAMPTZ )
+            .withDefault( "'-infinity'" )
```

Comment:
`'-infinity'` seems like a good candidate for a constant somewhere.

Reply:
Made a ticket for this as there's a few of these that should be constantified, and it'll involve finding/updating more files than just this one https://jira.openlattice.com/browse/LATTICE-2531